apps/site/components/EOL/VulnerabilityChips/index.tsx

```diff
@@ -1,9 +1,10 @@
 import type { FC } from 'react';
 
-import VulnerabilityChip from '#site/components/EOL/VulnerabilityChips/Chip';
 import { SEVERITY_ORDER } from '#site/next.constants.mjs';
 import type { Severity, Vulnerability } from '#site/types';
 
+import VulnerabilityChip from './VulnerabilityChip';
```

Comment:
looks like the refactor created some relative imports

Reply:
I think same level relative imports are OK, no? cc @nodejs/nodejs-website